src/map/battle.cpp

```diff
@@ -5358,6 +5358,12 @@ static struct Damage initialize_weapon_data(struct block_list *src, struct block
 		wd.flag |= battle_range_type(src, target, skill_id, skill_lv);
 		switch(skill_id)
 		{
+#ifdef RENEWAL
+			case RG_BACKSTAP:
+				if (skill_id == RG_BACKSTAP && sd && sd->status.weapon == W_DAGGER)
```

Comment:
```suggestion
				if (sd && sd->status.weapon == W_DAGGER)
```

Reply:
I'm still not sure whether it is 2 separate hits or just 200% damage showing 2 hits. In this pull request I assumed it is 200% in one hit and client shows 2 hit.

Reply:
From the GIF, and the kRO description `단검을 장착하여 사용할 경우 데미지가 2회 발생한다.` it would seem it sends 2 separate hits now, not double damage. So the solution is to remove the double damage.